src/mono/System.Private.CoreLib/src/System/ArgIterator.cs

```diff
@@ -59,7 +59,7 @@ public override int GetHashCode()
         public TypedReference GetNextArg()
         {
             if (num_args == next_arg)
-                throw new InvalidOperationException("Invalid iterator position.");
+                throw new InvalidOperationException(SR.InvalidOperation_InvalidIteratorPosition);
```

Comment:
```suggestion
                throw new InvalidOperationException(SR.InvalidOperation_EnumEnded);
```
Similar existing string